### PR TITLE
position angle

### PR DESCRIPTION
**Pull request by mileslucas**
Adds `position_angle` for getting the position angle between two coordinates.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -87,13 +81,39 @@ end
...
 
+# Test separation between coordinates and conversion with mixed floating types.
+@testset "Position Angles" begin
+    c1 = ICRSCoords(0, 0)
+    c2 = ICRSCoords(deg2rad(1), 0)
+
+    # interface
+    @inferred position_angle(c1, c2)
+    @inferred position_angle(c1, c2 |> GalCoords)
+    @test position_angle(c1, c2) == position_angle(c1, c2 |> GalCoords)
```
Comment (giordano) [inline test/runtests.jl]: Maybe you want to keep inference test separate from the actual test, but note that you can combine everything in a single line:
```suggestion
    @test @inferred(position_angle(c1, c2)) == @inferred(position_angle(c1, c2 |> GalCoords))
```
However, this is probably a matter of taste, not super important.

File: test/runtests.jl [hunk 2]
```diff
@@ -19,28 +19,22 @@ indata, inhdr = readdlm(fname, ','; header = true)
...
         end
 
-        for (outsys, S) in (
-            ("icrs", ICRSCoords{F}),
+        for (outsys, S) in (("icrs", ICRSCoords{F}),
             ("fk5j2000", FK5Coords{2000,F}),
             ("fk5j1975", FK5Coords{1975,F}),
-            ("gal", GalCoords{F}),
-        )
+            ("gal", GalCoords{F}),)
```
Comment (giordano) [inline test/runtests.jl]: Actually I think that the hanging parentheses here are better, they allow us to add more types or coordinates without touching other lines, useful to have a clean history.

Generally speaking, I'm not a big fan of touching lines without functional changes, these kinds of edits make using tools like `git blame` harder and the history more noisy and distracting.

**Comment by mileslucas** [comment]
Argh I keep forgetting to turn my autoformatter off for PRs. Let me try this again.

**Comment by mileslucas** [comment]
@giordano ready for review again.

**Pull request merged**